internal/monitoring/middlewares.go

```diff
@@ -14,6 +16,8 @@ const (
 	// IDPathRegex regexp used to swap the {id*} parameters in the path with simply id
 	// supports alphabetic characters and underscores, no dashes
 	IDPathRegex string = "{[a-zA-Z_]*}"
+	HTML        string = ".html"
+	UNKNOWN     string = "unknown"
```

Comment:
I think that this should be something more descriptive like (is `view` a better word?)
```suggestion
	UNKNOWN     string = "<unknown endpoint>"
```